arwen/host/arwen.go

```diff
@@ -311,69 +306,55 @@ func (host *vmHost) RunSmartContractCreate(input *vmcommon.ContractCreateInput)
 	host.mutExecution.RLock()
 	defer host.mutExecution.RUnlock()
 
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*2)
```

Comment:
might export a constant or even a config/flag for the 2 seconds value

Reply:
done